<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="img_fundo_login">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>850</width>
     <height>500</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(197, 68, 76);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="fundo_login.qrc">:/fundo_login/imagens/img_telaLogin.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextInteractionFlag::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="btn_esqueceu_senha">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Esqueceu a senha?</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_faca_login">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:700; color:#c38521;&quot;&gt;FAÇA LOGIN&lt;/span&gt;&lt;span style=&quot; font-size:28pt;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_entrar">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 600 9pt &quot;Segoe UI&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(197, 68, 76);
    border-radius: 10px;
}

QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: rgb(197, 68, 76);
}


</string>
   </property>
   <property name="text">
    <string>Entrar</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="btn_lembrar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>78</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Lembrar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>271</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="box_user">
    <item>
     <widget class="QLabel" name="icon_user">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="fundo_login.qrc">:/fundo_login/icons/user.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_user">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
      </property>
      <property name="placeholderText">
       <string>Digite seu usuário</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="box_senha">
    <item>
     <widget class="QLabel" name="icon_cadeado">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="fundo_login.qrc">:/fundo_login/icons/cadeado_p.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_senha">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Digite sua senha</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="fundo_login.qrc"/>
 </resources>
 <connections/>
</ui>
